test/e2e/resourceinterpreter_test.go

```diff
@@ -271,6 +280,23 @@ var _ = ginkgo.Describe("Resource interpreter webhook testing", func() {
 			})
 		})
 	})
+
+	ginkgo.Context("InterpreterOperation InterpretDependency testing", func() {
+		ginkgo.It("InterpretDependency testing", func() {
+			ginkgo.By("check if workload's dependency is interpreted", func() {
+				resourceBindingName := names.GenerateBindingName(cm.Kind, configMapName)
+
+				gomega.Eventually(func(g gomega.Gomega) (int, error) {
+					resourceBinding, err := karmadaClient.WorkV1alpha2().ResourceBindings(configMapNamespace).Get(context.TODO(), resourceBindingName, metav1.GetOptions{})
+					g.Expect(err).NotTo(gomega.HaveOccurred())
+
+					klog.Infof("Attached ResourceBinding(%s/%s)'s schedule result is %v, existing schedule result is expected.",
+						resourceBinding.Namespace, resourceBinding.Name, resourceBinding.Spec.RequiredBy)
+					return len(resourceBinding.Spec.RequiredBy), nil
+				}, pollTimeout, pollInterval).ShouldNot(gomega.BeZero())
+			})
+		})
+	})
```

Comment:
Maybe we can directly determine whether the cm is distributed to the target member cluster.

Reply:
Strictly speaking, we only need to check whether the dependent configmap is correctly interpreted, as for whether it is distributed to the relevant member clusters, I don't think it does care because the configmap is distributed is not the responsibility of the `InterpretDependency` operation. Maybe there is wrong with the `execution_controller` or `binding_controller`?

Reply:
For other operations, propagating cm is unnecessary because it consumes extra resources and may affect our analysis of e2e when it goes wrong.

Reply:
We should clearly judge the resources that follow the distribution. 

> Maybe we can directly determine whether the cm is distributed to the target member cluster.

The feature that follows the distribution is dependent on the two controllers you mentioned.